packages/web-ui/src/ds/tokens/skeleton.ts

```diff
@@ -0,0 +1,14 @@
+export const skeleton = {
+  height: {
+    h8: 'h-[8px]', // 8px
```

Comment:
Review always tailwind please. `h8` is `h-2` and `h7` is `h-2.5` 
https://tailwindcss.com/docs/height